src/applications/claims-status/containers/StemClaimStatusPage.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { setUpPage } from '../utils/page';
+
+import { Link } from 'react-router';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+import StemAskVAQuestions from '../components/StemAskVAQuestions';
+import ClaimsBreadcrumbs from '../components/ClaimsBreadcrumbs';
+import ClaimsUnavailable from '../components/ClaimsUnavailable';
+import StemDeniedDetails from '../components/StemDeniedDetails';
+
+class StemClaimStatusPage extends React.Component {
+  componentDidMount() {
+    this.setTitle();
+    setUpPage();
+  }
+
+  setTitle() {
+    document.title =
+      'Status - Your Edith Nourse Rogers STEM Scholarship Application Claim';
```

Comment:
This doesn't appear to match the `h1` content (`Your Edith Nourse Rogers STEM Scholarship Application`) on the details page. I don't remember where this is documented, but the `document.title` should match the `h1`.